benchmarks/benchmarks_for_parse_time.py

```diff
@@ -0,0 +1,24 @@
+# Initial benchmarks
+# See "Writing benchmarks" in the asv docs for more information.
```

Comment:
```suggestion
```

Reply:
I want to leave this "initial" comment cause this benchmarks are experimental, I don't know if they are good enough. So I want to mark them as "initial", only as examples to test ASV work. And when I'll write sets of benchmarks to cover sunpy modules, this two may be rewritten or deleted.

Reply:
I don't see the need to mark them. We know they are experimental.

Reply:
Good